package.json

```diff
@@ -6,9 +6,9 @@
     "test": "pulp test"
```

Comment:
CI is failing because of this line. We are using `npm run-script test --if-present`. NPM finds that there is such a script (above) and executes it. However, there aren't any tests in this repo. I'm assuming we should drop this 'test' script here?

Reply:
Yea, that works. Thanks!